### ISSUE TITLE
Incorrect biquads when exporting or using the post builder

### ISSUE THREAD

**Issue by bmiller** [3ll3d00d/beqdesigner#384]
When using the "Export BEQ Filter" or creating a file in the Post Builder incorrect values for the dec and hex keys are being written.  The biquads are being calculated when the filter is created using the signal's fs of 1000.  When either of the export features are used they are not being re-calculated to the device's fs of 96000.  

I'm not sure the cleanest fix for this.  I was going to recalculate them on export but I'm not sure if there would be any side effects with doing so since they're being used in the main window and other parts of the app.

**Comment by 3ll3d00d** [comment]
has it ever worked? I'm not sure it has :facepalm: 

converting on output is the way to go for sure, I can do this in the next day or two I think

**Comment by bmiller** [comment]
I don't know if it ever has.  I never use those features. I had always used the plug to create files since Aron was doing the BEQ design.  MOberhardt uses it and noticed that when I edited his files to add metadata that the biquads weren't matching.  It rewrites the filters since the app also has a filter editor.  I'm using it now to create the files for the repo.  The situation had me confused for a bit because I had validated my implementation against every file in a merged output and spot checked agains the plugin.  So it didn't make sense why we weren't matching.  That's when I found that the export gave different values and tracked it down.
